pkg/models/compatibility.go

```diff
@@ -8,3 +8,14 @@ type VMCompatibility struct {
 }
 
 type AvagoCompatiblity map[string][]string
+
+type NetworkVersion struct {
+	LatestVersion     string `json:"latest-version"`
+	RequirePrerelease bool   `json:"require-prerelease"`
```

Comment:
This is for case like https://github.com/ava-labs/avalanche-cli/pull/2674, where we had to use a prerelease for fuji only